regression-test/suites/nereids_rules_p0/mv/negative/negative_test.groovy

```diff
@@ -0,0 +1,633 @@
+    mtmv_sql_1 = """
+        select l_shipdate, l_partkey, l_orderkey, count(*) from lineitem_1 group by l_shipdate, l_partkey, l_orderkey
+        """
+    create_mv_lineitem(mv_name_1, mtmv_sql_1)
+    job_name_1 = getJobName(db, mv_name_1)
+    waitingMTMVTaskFinished(job_name_1)
+
+    query_sql_1 = """
+        select l_shipdate, l_partkey, l_orderkey, count(*) 
+        from lineitem_1 left join orders_1 
+        on lineitem_1.l_shipdate=orders_1.o_orderdate 
+        group by l_shipdate, l_partkey, l_orderkey
+        """
+    explain {
+        sql("${query_sql_1}")
+        notContains "${mv_name_1}(${mv_name_1})"
+    }
+
+    mtmv_sql_1 = """
+        select o_orderdate, o_shippriority, o_comment, l_orderkey, o_orderkey, sum(o_orderkey)   
+            from orders_1 
+            left join lineitem_1 on lineitem_1.l_orderkey = orders_1.o_orderkey
+            group by 
+            o_orderdate, 
+            o_shippriority, 
+            o_comment,
+            l_orderkey,
+            o_orderkey
+        """
+    create_mv_orders(mv_name_1, mtmv_sql_1)
+    job_name_1 = getJobName(db, mv_name_1)
+    waitingMTMVTaskFinished(job_name_1)
+
+    query_sql_1 = """
+        select o_orderdate, o_shippriority, o_comment, l_orderkey, ps_partkey, sum(o_orderkey)
+            from orders_1
+            left join lineitem_1 on lineitem_1.l_orderkey = orders_1.o_orderkey
+            left join partsupp_1 on partsupp_1.ps_partkey = lineitem_1.l_orderkey
+            group by
+            o_orderdate,
+            o_shippriority,
+            o_comment,
+            l_orderkey, 
+            ps_partkey
+        """
+    explain {
+        sql("${query_sql_1}")
+        notContains "${mv_name_1}(${mv_name_1})"
+    }
+
+    // union rewrite
+    mtmv_sql_1 = """
+        select l_shipdate, o_orderdate, l_partkey, count(*)
+        from lineitem_1
+        left join orders_1
+        on lineitem_1.l_orderkey = orders_1.o_orderkey
+        where l_shipdate >= "2023-10-17"
+        group by l_shipdate, o_orderdate, l_partkey
+        """
+    create_mv_lineitem(mv_name_1, mtmv_sql_1)
+    job_name_1 = getJobName(db, mv_name_1)
+    waitingMTMVTaskFinished(job_name_1)
+
+    query_sql_1 = """
+        select l_shipdate, o_orderdate, l_partkey, count(*)
+        from lineitem_1
+        left join orders_1
+        on lineitem_1.l_orderkey = orders_1.o_orderkey
+        where l_shipdate >= "2023-10-15"
+        group by l_shipdate, o_orderdate, l_partkey
+        """
+    explain {
+        sql("${query_sql_1}")
+        notContains "${mv_name_1}(${mv_name_1})"
+    }
+
+    mtmv_sql_1 = """
+        select l_shipdate, l_partkey, l_orderkey
+        from lineitem_1
+        where l_shipdate >= "2023-10-10"
+        group by l_shipdate, l_partkey, l_orderkey
+        """
+    create_mv_lineitem(mv_name_1, mtmv_sql_1)
+    job_name_1 = getJobName(db, mv_name_1)
+    waitingMTMVTaskFinished(job_name_1)
+
+    query_sql_1 = """
```

Comment:
why all name with same suffix `_1`

Reply:
It is not necessary and has been modified